Add option relativePath to write chunk paths relative from tracker-file path

melck opened the pull request:
Proposal for #25

veryhappythings commented on index.js:
```diff
@@ -55,14 +55,17 @@ Plugin.prototype.apply = function(compiler) {
       }
 
       var chunks = {};
+      var isRelativeRoot = this.options.path && !path.isAbsolute(this.options.path)
```
I'm relatively new to Javascript, so forgive me if this comment is misplaced, but I found that to get this to work I had to use `self` here and throughout these code changes instead of `this`. We use self below when checking for the `publicPath`, so I believe it's correct in the below context, but it also seems to be required here.

My changes are here https://github.com/veryhappythings/webpack-bundle-tracker/commit/8bb4d378f66ad6072a9db35fe3218aa13043533f and work for me - using Node v9.11.1

melck commented:
Thanks for reviewing, you are right about context. I have applied changes from your code

melck commented:
Fix behavior of relative paths like described in #25

mbark commented:
Are there any updates on this PR, it would be great to have :)

pySilver commented:
@owais @melck will it eventually get merged? :)

melck commented:
I have updated my merge request to modify behavior. I have changed this to make it optionnal. I have added doc and a test.

@owais Could you check and merge this ?

owais commented:
Thanks @melck
